data/zilliqa/assets/zil1w5hwupgc9rxyuyd742g2c9annwahugrx80fw9h/meta.json

```diff
@@ -2,9 +2,14 @@
   "name": "Gary",
   "symbol": "GARY",
   "web": "http://thegarytoken.com",
-  "supply": "100",
+  "supply": 100,
+  "decimals": 4,
+  "email": "thegarytoken@gmail.com",
+  "publicTeam": true,
```

Comment:
remove this, the team isnt public

Reply:
how can I prove the team is public - its just me and I have a public profile?

Reply:
what do you consider your public profile exactly? an empty github account named darthgus?